Cubical/Algebra/Group/Abelianization/Properties.agda

```diff
@@ -185,7 +177,7 @@ module _ (G : Group ℓ) where
                η (y · x)        ≡⟨ refl ⟩
                (η y) ·Ab (η x) ∎)
 
-  -- The proof that that the abelianization is in fact an abelian group.
+  -- The proof that the abelianization is in fact an abelian group.
   asAbelianGroup : AbGroup ℓ
   asAbelianGroup = makeAbGroup 1Ab _·Ab_ invAb isset assocAb ridAb rinvAb commAb
 
```

Comment:
Here you could end the module and add a definition 'AbelianizationAbGroup' after the module, so this name (which satisfies the new naming conventions) gets exported.

Reply:
added module and definition